data-prepper-plugins/aws-plugin/src/test/java/org/opensearch/dataprepper/plugins/aws/CredentialsProviderFactoryTest.java

```diff
@@ -99,6 +102,17 @@ void providerFromOptions_with_StsRoleArn() {
         assertThat(awsCredentialsProvider, instanceOf(StsAssumeRoleCredentialsProvider.class));
     }
 
+    @Test
+    void getDefaultRegion_returns_expected_region() {
+        when(defaultStsConfiguration.getAwsRegion()).thenReturn(Region.US_EAST_1);
```

Comment:
This is a good test to make parameterized on `Region` using `@EnumSource`. I think we have some other unit tests that select a handful of regions to verify against

Reply:
Region is not an enum it is a final class